src/main/logger.ts

```diff
@@ -1,6 +1,27 @@
 import { app, remote } from "electron";
 import winston from "winston"
 import { isDebugging } from "../common/vars";
+import { format } from "logform";
+import stringify from "string.ify";
+import { MESSAGE } from "triple-beam";
+
+const debugSimple = format(info => {
```

Comment:
Should not be part of this PR.

Reply:
I understand, however I really think that this should be part of 3.6.0 because when `DEBUG=true` is passed the application freezes whenever it prints out the cluster store json object.